FGEGraphics/UISystem/UIElementText.cs

```diff
@@ -32,24 +33,28 @@ public struct InternalData
         /// <summary>The raw string content of this text.</summary>
         public string Content;
 
-        /// <summary>Whether the text is empty and shouldn't be rendered.</summary>
-        public bool Empty;
-
         /// <summary>The maximum total width of this text, if any.</summary>
         public int MaxWidth;
 
+        /// <summary>A style/renderable pair internal to this text object.</summary>
+        public (UIElementStyle Style, RenderableText Text) InternalRenderable;
+
         /// <summary>A cache mapping a UI element's text styles to renderable text.</summary>
         public Dictionary<UIElementStyle, RenderableText> RenderableContent;
     }
 
+    // TODO: Get rid of text alignments
```

Comment:
get rid of? why?

Reply:
`UIAnchor` positions elements relative to the screen. Now we need a *draw origin* point relative to the *element*. This is implemented on `UIElementText` with text alignments right now, which is totally flawed. Shouldn't get rid of the text alignment enums themselves, just how I used them here.